src/main/java/moe/banana/jsonapi2/HasOne.java

```diff
@@ -19,6 +19,19 @@ public T get() throws ResourceNotFoundException {
         return (T) resource._doc.find(linkage);
     }
 
+    public T get(T defaultValue) {
+        try {
+            return get();
+        }
+        catch (ResourceNotFoundException e){
```

Comment:
It looks better in `} catch (ResourceNotFoundException e) {`, take care of the spaces.